### PR TITLE
GK-modify-read-functions

### LINKED ISSUE #9: Write read_man function
related to #4

### LINKED ISSUE #18: Write read_yld
related to #4

### PR DESCRIPTION
**Pull request by kgulzina**
Added two functions and one file: 

read_man(): 
- reads .man file
- uses supplementary Python code written by Daryl
- closes #9 

read_yld():
- outputs a list with data frames
- **documentation** needs a review
- closes #18 

read_cli():
- removed row names in breakpoints
- updated Dates
- closes issue #1

### CODE REVIEW

File: R/read_yld.R [hunk 1]
```diff
@@ -4,7 +4,64 @@
...
+      blank_line = 0
+
+      crop_type = as.numeric(tmp[6])
+      date = as.numeric(tmp[9])
+      OFE = as.numeric(tmp[13])
+      yield = as.numeric(tmp[18])
+      unit = tmp[19]
+      year = as.numeric(tmp[21])
+
+      row = c(crop_type, date, OFE, yield, unit, year)
```
Comment (jarad) [inline R/read_yld.R:50]: All variables are converted to character here.

Comment (jarad) [reply R/read_yld.R:50]: So I believe I shouldn't merge this until you fix the type conversion. Correct? @kgulzina

Comment (kgulzina) [reply R/read_yld.R:50]: yes, @jarad

Comment (kgulzina) [reply R/read_yld.R]: @jarad, I fixed the type conversion. Can merge now.

**Comment by jarad** [comment]
Do we need to add `reticulate` as a Depends/Imports/Suggests? And then do we need to determine the python version required?

**Review by jarad — APPROVED** [review]
Well I didn't test this but let's merge it. Probably want to consider some unit tests in the future.

**Pull request merged**